website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -37,6 +37,144 @@ kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef
 WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE THAT YOU COPY THE BETA API ALERT SECTION FROM THE LAST RELEASE TO PROPERLY WARN USERS OF THE RISK OF UPGRADING WITHOUT GOING TO 0.32.x FIRST
 -->
 
+### Upgrading to `1.0.0`+
```

Comment:
I don't see us providing any detail on the API changes that were made in v1. Do we want to make a callout about the specific parts of the API that changed so that users can update their manifests that they may have written locally

Reply:
With the new "v1 Migration Guide" I kept the "Changelog" section of the previous migration guide. So far, I have just dropped in the API changes that I had described earlier based on when then needed to be included: before v1.0 upgrade or before v1.1 upgrade. Maybe we should discuss if we want to cover this content in a different way.